### ISSUE TITLE
[Feature]: Allow compact version of PowerApps Editable Grid

### ISSUE THREAD

**Issue by nstratton-srx** [microsoft/fluentui#33474]
### Area

React Components (@fluentui/react-components)

### Describe the feature that you would like added

The underlying PAEG has spacing and compact options to increase the number of rows displayed. The current grids have an excess of padding that reduces the number of rows one can show.

Can the grid wrapper pass through spacing and compact settings? This would be greatly appreciated.

### Have you discussed this feature with our team

@ling1726

### Validations

- [x] Check that there isn't already an issue that requests the same feature to avoid creating a duplicate.

### Priority

None

**Comment by layershifter** [comment]
> The underlying PAEG has spacing and compact options to increase the number of rows displayed. The current grids have an excess of padding that reduces the number of rows one can show.
> 
> Can the grid wrapper pass through spacing and compact settings? This would be greatly appreciated.

@nstratton-srx not that it's clear enough, do you mind to provide more details? Screenshots, examples, references?

**Comment by nstratton-srx** [comment]
Hello,

I was slightly vague in my post, as you are using the third-party AG Grid under the hood. The most relevant would be to show examples of how AG greatly increase their information density, but I was hesitant to show that on your post. I could show the AG gird as a “sample” reference without hinting that’s what’s under the hood. Your thoughts?

Kind regards,

Nigel Stratton
Senior Software Developer

[cid:shred-xlogonosd_7dc1a28e-1703-4954-a349-dd50da382e99.png]
 p. 1300 747 339
 shred-x.com.au<http://shred-x.com.au/>
 [cid:Facebooklogo24x24_a6e2a514-163d-44ac-b157-7a0b899c16cd.png] <https://www.facebook.com/ShredXSecureDestructionAustralia> <https://www.facebook.com/ShredXSecureDestructionAustralia>  [cid:LinkedInlogo24x24v1_39ac0ec0-d2d5-4a26-859f-f9262e0f1036.png] <https://www.linkedin.com/company/shred-x-pty-ltd> <https://www.linkedin.com/company/shred-x-pty-ltd> <https://www.linkedin.com/company/shred-x-pty-ltd>

<https://www.shred-x.com.au/e-waste/><https://www.shred-x.com.au/><https://www.shred-x.com.au/document-destruction/office-cleanouts-relocations/><https://www.shred-x.com.au/document-destruction/office-cleanouts-relocations/><https://www.shred-x.com.au/eofy-2024/><https://www.shred-x.com.au/><https://www.shred-x.com.au/2024-spring-promotion/><https://www.shred-x.com.au/contact/><https://www.shred-x.com.au/circular-economy-2/><https://www.shred-x.com.au/>[cid:christmasemailbanner2024_af590fc0-524d-413c-968c-c06b9688afc4.png]
<https://www.shred-x.com.au/holiday-season/><https://www.shred-x.com.au/circular-economy-2/><https://www.shred-x.com.au/holiday-season/><https://www.shred-x.com.au/>

<https://www.shred-x.com.au/data-destruction/><https://www.shred-x.com.au/data-destruction/><https://www.shred-x.com.au/data-destruction/><https://www.shred-x.com.au/data-destruction/><https://www.shred-x.com.au/data-destruction/>

<https://www.shred-x.com.au/>

<https://www.shred-x.com.au/holiday-season/><https://www.wasteexpoaustralia.com.au/en-gb/conference.html><https://www.shred-x.com.au/holiday-season/><https://www.shred-x.com.au/circular-economy-2/><https://www.shred-x.com.au/holiday-season/><https://www.shred-x.com.au/>

<https://www.shred-x.com.au/>

<https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://events.dell.com/event/341ec44c-2ca6-4a89-a679-a14eb503ba5c/summary><https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/holiday-season/><https://www.wasteexpoaustralia.com.au/en-gb/conference.html><https://www.shred-x.com.au/holiday-season/><https://www.shred-x.com.au/circular-economy-2/><https://www.shred-x.com.au/holiday-season/><https://www.shred-x.com.au/>

<https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/holiday-season/><http://www.shred-x.com.au/contact/#contact-tab|1><https://www.shred-x.com.au/easter-2023/><https://www.shred-x.com.au/why-shred-and-destroy/><https://www.shred-x.com.au/e-waste/><https://www.shred-x.com.au/eofy-2023/><https://www.shred-x.com.au/><https://events.dell.com/event/341ec44c-2ca6-4a89-a679-a14eb503ba5c/summary><https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/holiday-season/><https://www.wasteexpoaustralia.com.au/en-gb/conference.html><https://www.shred-x.com.au/holiday-season/><https://www.shred-x.com.au/circular-economy-2/><https://www.shred-x.com.au/holiday-season/><https://www.shred-x.com.au/><https://www.shred-x.com.au/>

<https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/holiday-season/><https://www.wasteexpoaustralia.com.au/en-gb/conference.html><https://www.shred-x.com.au/holiday-season/><https://www.shred-x.com.au/circular-economy-2/><https://www.shred-x.com.au/holiday-season/><https://www.shred-x.com.au/>

<https://www.shred-x.com.au/><https://events.dell.com/event/341ec44c-2ca6-4a89-a679-a14eb503ba5c/summary><https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/><https://www.shred-x.com.au/holiday-season/><https://www.wasteexpoaustralia.com.au/en-gb/conference.html><https://www.shred-x.com.au/holiday-season/><https://www.shred-x.com.au/circular-economy-2/><https://www.shred-x.com.au/holiday-season/><https://www.shred-x.com.au/>

OUR SERVICES:
<https://www.shred-x.com.au/document-destruction/>DOCUMENT DESTRUCTION<https://www.shred-x.com.au/document-destruction/>  |  <https://www.shred-x.com.au/e-waste/> E-WASTE & IT ASSET LIFECYCLE SOLUTIONS<https://www.shred-x.com.au/e-waste/>  |  <https://www.shred-x.com.au/product-uniform-destruction/> PRODUCTS & TEXTILES<https://www.shred-x.com.au/product-uniform-destruction/>  |  <https://www.shred-x.com.au/circular-economy-2/> WASTE RENEWAL<https://www.shred-x.com.au/circular-economy-2/>

<https://www.shred-x.com.au/><https://www.shred-x.com.au/holiday-season/>

Caution: The information contained in this e-mail is confidential and may be legally privileged.
If the reader of this message is not the intended recipient you are hereby notified that any use, dissemination, distribution, or reproduction of this message is prohibited.
If you have received this message in error please reply to the sender and delete all copies of this transmission.
Although Shred-X has taken every precaution to ensure no viruses are present in this email, the company cannot accept responsibility for any loss or damage arising from the use of this email or attachments.

Please consider the environment before printing this email. Shred-X saves thousands of trees annually by recycling<http://www.shred-x.com.au/our-environment?utm_source=Email&utm_medium=Email&utm_campaign=Signature> our clients paper that we collect and process.

**Comment by nstratton-srx** [comment]
Looking at the PAEG in a form, there is significant padding in the grid, resulting in a low row density. This is a constant source of frustration and requires we license an expensive grid that offers high density rows.

![Image](https://github.com/user-attachments/assets/a57b5f66-25bc-4484-8c28-ec297933d72d)

In this version of the grid (the same underlying AG grid) we see a high density of rows.

![Image](https://github.com/user-attachments/assets/5f62fad9-7760-4580-ab16-88648da19d45)

In the second picture, the grid's theme has several options to increase row density, including spacing, rowHeight, headerHeight, listItemHeight. Please expose these options to allow a higher row density.

**Comment by strattonn** [comment]
Was that sufficient? The huge amount of wasted space is simply a horrendous addition to nice looking pages.

![Image](https://github.com/user-attachments/assets/6e741363-2f2b-4369-8a97-6098e2a708a1)

**Comment by layershifter** [comment]
@ling1726 can you please check?

**Comment by ling1726** [comment]
This feature request seems reasonable, we support different sizing for other components, that we could apply similarly to elements in the grid.

@gouttierre  can you please help triage this feature request?

**Comment by strattonn** [comment]
Certainly. This would would help us greatly in many places. What would you like me to do?

**Comment by nstratton-srx** [comment]
Keep open. This would be very helpful. Let me know what I can do to help.
